packages/std/src/results/events.rs

```diff
@@ -62,6 +62,48 @@ impl Attribute {
     }
 }
 
+impl<K: Into<String>, V: Into<String>> From<(K, V)> for Attribute {
+    fn from((k, v): (K, V)) -> Self {
+        Attribute::new(k, v)
+    }
+}
+
+impl<K: AsRef<str>, V: AsRef<str>> PartialEq<(K, V)> for Attribute {
```

Comment:
Ah, no these... it would help to add a comment about these for making test functions simpler.
And document their usage in MIGRATING.md... or maybe just some good rustdocs on Attribute / Response.

Reply:
I can do that.